cartography/intel/aws/ec2/instances.py

```diff
@@ -21,410 +29,343 @@
-                launch_time_unix = str(time.mktime(launch_time.timetuple()))
-            else:
-                launch_time_unix = ""
-
-            neo4j_session.write_transaction(
-                _load_ec2_instance_tx,
-                instanceid,
-                instance,
-                reservation_id,
-                monitoring_state,
-                launch_time,
-                launch_time_unix,
-                instance_state,
-                current_aws_account_id,
-                region,
-                update_tag,
-            )
-
-            # SubnetId can return None intermittently so attach only if non-None.
-            subnet_id = instance.get('SubnetId')
-            if subnet_id:
-                neo4j_session.write_transaction(_load_ec2_subnet_tx, instanceid, subnet_id, region, update_tag)
-
-            if instance.get("KeyName"):
-                key_name = instance["KeyName"]
-                key_pair_arn = f'arn:aws:ec2:{region}:{current_aws_account_id}:key-pair/{key_name}'
-                neo4j_session.write_transaction(
-                    _load_ec2_keypairs_tx,
-                    key_pair_arn,
-                    key_name,
-                    region,
-                    instanceid,
-                    current_aws_account_id,
-                    update_tag,
-                )
-
-            if instance.get("SecurityGroups"):
-                for group in instance["SecurityGroups"]:
-                    group_id = group["GroupId"]
-                    neo4j_session.write_transaction(
-                        _load_ec2_security_groups_tx,
-                        group_id,
-                        group,
-                        instanceid,
-                        region,
-                        current_aws_account_id,
-                        update_tag,
-                    )
-
-            load_ec2_instance_network_interfaces(neo4j_session, instance, update_tag)
-            instance_ebs_volumes_list = get_ec2_instance_ebs_volumes(instance)
-            load_ec2_instance_ebs_volumes(neo4j_session, instance_ebs_volumes_list, current_aws_account_id, update_tag)
+    load(
+        neo4j_session,
+        EC2NetworkInterfaceSchema(),
+        network_interface_list,
+        Region=region,
+        AWS_ID=current_aws_account_id,
+        lastupdated=update_tag,
+    )
 
 
 @timeit
-def get_ec2_instance_ebs_volumes(instance: Dict) -> List[Dict]:
-    instance_ebs_volumes_list: List[Dict] = []
-    if 'BlockDeviceMappings' in instance and len(instance['BlockDeviceMappings']) > 0:
-        for mapping in instance['BlockDeviceMappings']:
-            if 'VolumeId' in mapping['Ebs']:
-                mapping['InstanceId'] = instance["InstanceId"]
-                instance_ebs_volumes_list.append(mapping)
-    return instance_ebs_volumes_list
-
-
-def _load_ec2_instance_ebs_tx(
-        tx: neo4j.Transaction,
-        ebs_data: List[Dict[str, Any]],
-        update_tag: int,
+def load_ec2_instance_nodes(
```

Comment:
copy pasta. duplicated at `load_instances()`

Reply:
Damn, good catch thank you